modules/nw-enabling-multicast.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * networking/using-multicast.adoc
+
+[id="enabling-multicast-{context}"]
+= Enabling multicast between pods
+
+You can enable multicast between pods for your project.
+
+.Prerequisites
+
+* The `oc` CLI program installed.
+
+.Procedure
+
+. Run the following command:
```

Comment:
Same about using a regular bullet if it's just one step

Reply:
World's simplest procedure!